application/models/classification_type.py

```diff
@@ -0,0 +1,13 @@
+from application.utils.database import db
+
+
+class ClassificationType(db.Model):
+    __tableargs__ = {"schema": "templatesvc"}
```

Comment:
I think this should be `__table_args__`

Reply:
http://docs.sqlalchemy.org/en/latest/orm/extensions/declarative/table_config.html